### ISSUE TITLE
infinite loop for pytorch and numpy standalone build

### ISSUE THREAD

**Issue by intgogo** [Nuitka/Nuitka#701]
Nuitka: 0.6.8rc7
Python: 3.7.4 (default, Aug 13 2019, 20:35:49) 
Executable: /root/lilw/anaconda3/envs/pytorch/bin/python
OS: Linux
Arch: x86_64
Cmd:
`python -m nuitka --show-progress --plugin-enable=numpy --plugin-enable=torch --show-scons -j 20  --follow-imports   --nofollow-import-to=matplotlib  --nofollow-import-to=sklearn --standalone main.py`
Result:
Infinite print... 
```
...
Nuitka:INFO:Optimizing module '_struct', 10 more modules to go after that.
Nuitka:INFO:Optimizing module '_opcode', 9 more modules to go after that.
Nuitka:INFO:Optimizing module '_multibytecodec', 8 more modules to go after that.
Nuitka:INFO:Optimizing module '_heapq', 7 more modules to go after that.
Nuitka:INFO:Optimizing module '_codecs_tw', 6 more modules to go after that.
Nuitka:INFO:Optimizing module '_codecs_kr', 5 more modules to go after that.
Nuitka:INFO:Optimizing module '_codecs_jp', 4 more modules to go after that.
Nuitka:INFO:Optimizing module '_codecs_iso2022', 3 more modules to go after that.
Nuitka:INFO:Optimizing module '_codecs_hk', 2 more modules to go after that.
Nuitka:INFO:Optimizing module '_codecs_cn', 1 more modules to go after that.
Nuitka:INFO:Optimizing module '_bz2', 0 more modules to go after that.
...
```

**Comment by melandur** [comment]
Encountered the same infinite loop error on windows.
Without the --standalone flag everything works as expected.

### Version
Nuitka 0.6.8.4
Python: 3.7.6 (default, Jan  8 2020, 20:23:39) [MSC v.1916 64 bit (AMD64)]
Executable: C:\Miniconda3\envs\deepbratumia\python.exe
OS: Windows
Arch: x86_64

### Cmd
python -m nuitka --standalone  --plugin-enable=numpy --plugin-enable=torch  --follow-imports --show-progress --show-scons main.py

### Code
    import torch
    
    if __name__ == '__main__':
        device = torch.device('cpu')
        # device = torch.device('cuda') # Uncomment this to run on GPU

        # N is batch size; D_in is input dimension;
        # H is hidden dimension; D_out is output dimension.
        N, D_in, H, D_out = 64, 1000, 100, 10

        # Create random input and output data
        x = torch.randn(N, D_in, device=device)
        y = torch.randn(N, D_out, device=device)

        # Randomly initialize weights
        w1 = torch.randn(D_in, H, device=device)
        w2 = torch.randn(H, D_out, device=device)

        learning_rate = 1e-6
        for t in range(500):
          # Forward pass: compute predicted y
          h = x.mm(w1)
          h_relu = h.clamp(min=0)
          y_pred = h_relu.mm(w2)

          # Compute and print loss; loss is a scalar, and is stored in a PyTorch Tensor
          # of shape (); we can get its value as a Python number with loss.item().
          loss = (y_pred - y).pow(2).sum()
          print(t, loss.item())

          # Backprop to compute gradients of w1 and w2 with respect to loss
          grad_y_pred = 2.0 * (y_pred - y)
          grad_w2 = h_relu.t().mm(grad_y_pred)
          grad_h_relu = grad_y_pred.mm(w2.t())
          grad_h = grad_h_relu.clone()
          grad_h[h < 0] = 0
          grad_w1 = x.t().mm(grad_h)

         # Update weights using gradient descent
          w1 -= learning_rate * grad_w1
          w2 -= learning_rate * grad_w2
